### PR TITLE
Updated the example for active and selected cells to include border s…

### PR DESCRIPTION
**Pull request by ann-marie-ward**
…tyle

Post-merge checklist:

The master branch is auto-deployed to `dash.plotly.com`.
Once you have merged your PR, wait 5-10 minutes and check dash.plotly.com
to verify that your changes have been made.

- [ ] I understand

If this PR documents a new feature of Dash:

- [ ] Comment on the original Dash issue with a link to the new docs.
- [ ] Reply to any community thread(s) asking for this feature.

If this PR includes a new dataset available at a remote URL:
- [ ] I have added this dataset to the `datasets/` folder
- [ ] I have added a mapping between the remote URL and the filename in the
`datasets/` folder into the `find_and_replace` dict in `dash_docs/tools.py`

If this PR adds an image or animated GIF:
- [ ] This image was saved and referenced locally rather than via an external link

If I introduced a new relative link inside `dcc.Markdown`:
- [ ] I considered whether I could replace the `dcc.Markdown` call with `rc.Markdown`, which will replace relative links with `tools.relpath` internally. Otherwise, I used e.g. `<dccLink href=tools.relpath('/layout') children="the first chapter"/>` instead of `[the first chapter](/layout)` (importing `tools` from `dash_docs`), and set `dangerously_allow_html=true` in the `dcc.Markdown` call.

If I changed the `chapter_index` by removing or relocating a page:
- [ ] I added a redirect in `dash_docs/server.py` from the old URL to the new URL

### CODE REVIEW

**Comment by ann-marie-ward** [comment]
Hi everyone.  This is my first ever PR, so please let me know if I did things wrong.  Thanks!

**Comment by chriddyp** [comment]
Welcome @ann-marie-ward ! Great to see the PR 😺  I'll take a look a little bit later on today 👍

File: dash_docs/chapters/dash_datatable/conditional_formatting/index.py [hunk 1]
```diff
@@ -146,6 +146,7 @@
                         'state': 'active'  # 'active' | 'selected'
                     },
                    'backgroundColor': 'rgba(0, 116, 217, 0.3)'
```
Comment (jdamiba) [inline dash_docs/chapters/dash_datatable/conditional_formatting/index.py:148]:
```suggestion
                   'backgroundColor': 'rgba(0, 116, 217, 0.3)',
```

**Comment by jdamiba** [comment]
Hi @ann-marie-ward! Sorry it has taken us a bit to get back to you- it's been a busy few weeks at Plotly HQ. 

This PR is currently failing our test suite, and I think I know why- if you would be so kind as to commit my suggestion, I think we should be able to get this merged. 

Thanks for using Dash!

**Comment by jdamiba** [comment]
@chriddyp I also need a code-owner to approve this PR. It looks good to me. 💃

**Comment by ann-marie-ward** [comment]
sorry for the typo - I'll do better next time.  Thanks! 😊

**Comment by jdamiba** [comment]
@ann-marie-ward I have approved the Percy image snapshots that were generated for this PR. Feel free to merge anytime!

**Pull request merged**